lua/autorun/server/owoifier.lua

```diff
@@ -6,12 +6,12 @@ local function OwOifier(text)
 	local splitstring = string.Split(text, "")
 	local isemoteenabled = false
 
-	if string.find("!/", splitstring[1]) then
```

Comment:
Why change this?

Reply:
The arguments were backwards. The first one is a "haystack" the second one is the "needle". All the errors were due to the input string using patterns when they're not supposed to. It still works, I tested it.

Reply:
Oh I see. Would need to do nopatterns=true to have the charset as the haystack